pages/early-access.tsx

```diff
@@ -10,10 +10,10 @@ const earlyaccess = () => {
          <MetaTag tagName='Early Access'/>
          <DesktopNavbar></DesktopNavbar>
          <Container>
-            <div className="md:h-[70vh] pt-40 relative flex items-center flex-col">
+            <div className="md:h-full pt-40 relative flex items-center flex-col">
                <AccessSection/>
             </div>
-            <Footer></Footer>
+            <Footer paddingTop="pt-50"></Footer>
```

Comment:
@maganijnr why add a paddingTop property to a component? Use the tailwindcss classname and the `className` props instead of `paddingTop`.

e.g
```tsx
<Footer className='pt-24'> 
```
instead of 
```tsx
<Footer paddingTop={24}>
```

I believe the approach you've taken is very limiting.